hello-security/login-controller.php

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Login</title>
+</head>
+<body>
+<a href="index.php"> Take me home, country roads ... </a> <br>
+<?php
+include "db-connection.php";
+$conn = openCon();
+
+$email = htmlentities($_POST["email"]);
+$password = htmlentities($_POST["password"]);
+
+//prepared statement - defence against SQL injection
+$stmt = $conn->prepare("SELECT * from person where email = :email");
```

Comment:
Maybe it's a good idea to extract sql query in separate variable to make the code more readable.